steps/step_0_corsika_py3_v4_1_0_branch_TrimShower.py

```diff
@@ -147,6 +150,18 @@ def main(cfg, run_number, scratch):
             **cfg['BaseSimulationBiasSettings']
         )
 
+
+    # -----------------------------------
+    # Delete keys 
```

Comment:
we can make this more general and flexible by simply adding a list of keys to delete.